lib/stylesheet/importer.rb

```diff
@@ -144,9 +144,8 @@ def category_css(category)
     end
 
     def to_scss_variable(name, value)
-      escaped = value.to_s.gsub('"', "\\22")
-      escaped.gsub!("\n", "\\A")
-      "$#{name}: unquote(\"#{escaped}\");\n"
+      escaped = SassC::Script::Value::String.quote(value, sass: true)
```

Comment:
Do we need to do this with other parts of the import process that has the potential to need escaping?
In other words, do we need do this at the top of the import funnel?

Reply:
Which "import funnel"? As far as I know, this is the only place that we inject user input directly into SCSS variables. If there is somewhere else, then yes we should use this escaping method.

Reply:
> As far as I know, this is the only place that we inject user input directly into SCSS variables.

👍 My bad, I thought that there will be other places within the stylesheet importer where we need to escaping.